docs/src/guide/index.md

```diff
@@ -677,6 +677,28 @@ If the provided capability has no invoke method, the return value here functions
 as `getCapability` including returning `undefined` if the capability is not 
 exposed.
 
+### Identifier Syntax
+
+For most purposes, a domain object identifier can be treated as a purely
+symbolic string; these are typically generated by Open MCT Web and plug-ins
+should rarely be concerned with its internal structure.
+
+A domain object identifier has one or two parts, separated by a colon.
+
+* If two parts are present, the part before the colon refers to the space
+  in which the domain object resides. This may be a persistence space or
+  a purely symbolic space recognized by a specific model provider.
+* If only one part is present, the domain object has no space specified,
+  and may presume to reside in the application-configured default space
+  defined by the `PERSISTENCE_SPACE` constant.
+
+```bnf
```

Comment:
Heh, this is a particular weakpoint of mine, but how many people can read BNF?  It seems there are other ways of specifying this that are much easier to comprehend.  An example would be very helpful here.